src/patternfly/components/Toolbar/toolbar-item.hbs

```diff
@@ -2,6 +2,9 @@
   {{#if toolbar-item--id}}
     id="{{toolbar-item--id}}"
   {{/if}}
+  {{#if toolbar-item--IsRelative}}
+    style="position: relative;"
+  {{/if}}
```

Comment:
Same here about this local style - can it be moved to local demo styles if it's necessary? Also could we just not show expanded menus in core since the tools used to position those no longer lives in the CSS?